apa102/src/main/java/com/google/android/things/contrib/driver/apa102/Apa102.java

```diff
@@ -67,7 +67,8 @@
     private Mode mLedMode;
 
     // RGB LED strip settings that have sensible defaults.
-    private int mLedBrightness = MAX_BRIGHTNESS >> 1; // default to half
+    private int mLedBrightnessGlobal = MAX_BRIGHTNESS >> 1; // Default to half
+    private int mLedBrightness[];
```

Comment:
best practice dictates to put the array initialiser on the field type, so you can easily see the difference between a single int and an int array.

`private int[] mLedBrightness;`

Reply:
good point, thanks!